src/ui/layout/mct-tree.vue

```diff
@@ -1,9 +1,13 @@
 <template>
 <div
+    ref="searchAndTree"
```

Comment:
If its the root element of the component, we can use this.$el, no need to add a ref

Reply:
good call! updated.